src/main/java/com/google/api/codegen/GeneratorBuilderUtil.java

```diff
@@ -14,13 +14,6 @@
  */
```

Comment:
I'll take care of that, and additionally I think I'll rename the class while I'm at it.